vcd/resource_vcd_nsxt_edgegateway.go

```diff
@@ -0,0 +1,414 @@
+package vcd
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"strings"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/vmware/go-vcloud-director/v2/govcd"
+	"github.com/vmware/go-vcloud-director/v2/types/v56"
+)
+
+var nsxtEdgeSubnetRange = &schema.Resource{
+	Schema: map[string]*schema.Schema{
+		"start_address": {
+			Required: true,
+			Type:     schema.TypeString,
+		},
+		"end_address": {
+			Required: true,
+			Type:     schema.TypeString,
+		},
+	},
+}
+
+var nsxtEdgeSubnet = &schema.Resource{
+	Schema: map[string]*schema.Schema{
+		"gateway": {
```

Comment:
for easier readability, would like to follow same order of fields, type, required, optional, descripction, etc..

Reply:
Fixed.